### ISSUE TITLE
Compile files due to failed download of tbb-VERSION tar distro

### ISSUE THREAD

**Issue by nadyawilliams** [COMBINE-lab/salmon#496]
**Describe the bug**
The compilation fails when prerequisite external sources are downloaded from 
the GitHub and the  downloaded file sha256 signature is compared to what is 
stored in salmon distribution  files.  The relevant portion of log:

make[6]: Entering directory `/export/repositories/biotools-admix/BUILD/salmon_1.1.0-1.1.0/salmon-1.1.0/build'
[  7%] Performing download step for 'libtbb'
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   125  100   125    0     0    976      0 --:--:-- --:--:-- --:--:--   976
100   126  100   126    0     0    787      0 --:--:-- --:--:-- --:--:--   787
100 2519k    0 2519k    0     0  3504k      0 --:--:-- --:--:-- --:--:-- 4772k
tbb-2019_U8.tgz: FAILED
sha256sum: WARNING: 1 computed checksum did NOT match

**To Reproduce**
Compiling with all needed prerequisites (outside of salmon) loaded via modules. 
Using Centos 7.7 with  gcc 8.4.0, boost 1.66.9

mkdir build
cd build
cmake -DBOOST_ROOT=$BOOST_HOME -DCMAKE_INSTALL_PREFIX=/opt/apps/salmon/1.1.0 .. 
make 

  * Which version of salmon was used?
salmon 1.1.9 (fails to download tbb-2019-U8.tgz)
salmon 0.14.0 (fails to download tbb-2018-U3.tgz)
 
The problem lies in the repository for tbb. At some recent point after March 5
( I have successfully built salmon 0.14.0 on March 5 with above tbb just fine), the tar balls
in tbb repo have changed and so changed their sha256sum signatures. 
The distro contents  (comparing old vs. new) appear to be the same with the exception 
of renaming of the top most directory from tbb-VERSION/ to oneTBB-VERSION/. 
When a tar ball is download its signature is different from what salmon expects in its files. 

**Additional context**
The fix would be to update salmon source distributions to have new sha256 signatures and
and other changes (top dir name) in Cmakefiles

**Comment by rob-p** [comment]
Dear @nadyawilliams,

  Indeed; we noticed this change recently as well.  Unfortunately, we expect the tagged versions of salmon to remain stable, so changing those is problematic.  However, we've already added the changes you suggest above on the develop branch so that v1.2.0 will not suffer this issue when built from source.  The relevant changes to CMakeLists.txt are actually quite small, so if you wish to apply them to a previous source build, it should be fairly easy.  Otherwise, if you want to work with the latest, develop should build cleanly pulling in the TBB libraries (and 1.2.0 should be out soon moving these changes into master).  Thanks again for raising this issue!
